### PR TITLE
Update Magento_AmqpStore ReadMe

### PR DESCRIPTION
**Pull request by diazwatson**
### Description (*)

Updates ReadMe for the Magento_AmqpStore module.

### Fixed Issues (if relevant)

1. magento/devdocs https://github.com/magento/devdocs/issues/4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/AmqpStore/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-# Amqp Store
+# Magento_AmqpStore module
 
-**AmqpStore** provides ability to specify store before publish messages with Amqp.
+Magento_AmqpStore provides ability to specify store before publish messages with with the Advanced Message Queuing Protocol (AMQP).
```
Comment (jeff-matthews) [inline app/code/Magento/AmqpStore/README.md]:
```suggestion
Magento_AmqpStore provides the ability to specify a store before publishing messages with the Advanced Message Queuing Protocol (AMQP).
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5707](https://jira.corp.magento.com/browse/ENGCOM-5707) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Pull request merged**